rq/queue.py

```diff
@@ -325,9 +325,6 @@ def create_job(self, func, args=None, kwargs=None, timeout=None,
             job.retries_left = retry.max
             job.retry_intervals = retry.intervals
 
-        if isinstance(depends_on, Dependency):
```

Comment:
I took the liberty and removed these lines as this check is already done inside job.create.